### PR TITLE
feat(web-modeler): add monorepo to git sync

### PR DESCRIPTION
**Pull request by JacobiCamunda**
## Description

Expanded documentation to support the newly added monorepo functionality to GitHub Sync.
See https://github.com/camunda/web-modeler/issues/10884

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] ~My changes are for an **already released minor** and are in `/versioned_docs` directory.~
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by JacobiCamunda** [comment]
@HanselIdes, please look at the changes. I did not add documentation to the CI/CD section yet as I am unsure about the scope after the read-only user story was put out of scope. I assume this [doc](https://docs.camunda.io/docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd/) is meant to be updated? Could you give me a bit more input for the adjustments you want to see here? I could only think about a small part about being able to catch an event in GitHub if Web Modeler pushed changes to a repository.

File: docs/components/modeler/web-modeler/git-sync.md [hunk 1]
```diff
@@ -49,10 +49,13 @@ An organization administration account (or project administrator in Camunda Self
    - **Installation ID:** Found in the URL of your GitHub App's installation page.
    - **Client ID:** Found in your GitHub App's settings page.
    - **Private Key:** The contents of the .pem file downloaded from your GitHub App's settings page.
-   - **GitHub repository URL:** The URL of the repository you would like to sync with.
+   - **GitHub repository URL:** The URL of the repository you would like to sync with. It must not contain the `.git` extension nor the path to a folder.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/git-sync.md:52]: Should this be the base URL? An example may help

File: docs/components/modeler/web-modeler/git-sync.md [hunk 2]
```diff
@@ -87,6 +90,32 @@ Existing GitHub configurations can be edited from the gear icon beside the **Syn
...
+
+Git sync supports parallel development of features by allowing multiple process applications to be connected to different feature branches. This enables teams to work on multiple features simultaneously without interfering with each other's work.
+
+To use Git sync for parallel feature development, follow these steps:
+
+1. Create a new process application in the Modeler for each active feature branch you wish to develop.
+2. Configure Git sync for each instance by connecting it to the corresponding feature branch in your repository.
+3. Work on your feature within the Modeler, and use the Sync with GitHub button to pull and push changes as needed.
+4. Once the feature is complete and merged into the main branch, the process application associated with the feature branch can be deleted.
+
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/git-sync.md:118]:
```suggestion

To perform hotfixes or patches of production or production-bound processes, sync a copy of a process application to the `main` branch.

```

**Review by HanselIdes — DISMISSED** [review]
@JacobiCamunda LGTM! I added a few non-blocking comments. You do not need to re-request my review.

For the CI/CD page, after looking over it again I think we need to work with the fine folks at DevEx to rewrite it completely. There's a lot of old content that requires some API choreography rather than the new features we built lately

**Comment by JacobiCamunda** [comment]
This is another comment from Eric from [Slack](https://camunda.slack.com/archives/C07NYQAED39/p1730279404168499):

> FYI for the docs, having multiple copies of a process application will cause problems. You will have multiple files with the same ID in the same project, which complicates navigation and deployment. To avoid this, users can put copies of the same process app in different projects. This can be accomplished with a warning, but Dev Ex would best know how to describe this problem

@camunda/tech-writers, as @HanselIdes wrote, I would rely on you to come up with a good wording for this problem and mitigation 🙂 

___

Regarding:

> For the CI/CD page, after looking over it again I think we need to work with the fine folks at DevEx to rewrite it completely. There's a lot of old content that requires some API choreography rather than the new features we built lately

@marcellobarile, should we take this with us as part of the GitLab Epic or a follow-up to revamp it together with DevEx?

**Comment by JacobiCamunda** [comment]
@camunda/tech-writers, could you add your review/work (see [comment](https://github.com/camunda/camunda-docs/pull/4530#issuecomment-2447114806)) here so we can be sure to get it in the docs release?

**Comment by mesellings** [comment]
Hi @JacobiCamunda, just taking a look at the scope of this - can I just confirm what your requirements are please?:
* A TW review of this specific PR, BUT also to include additional work reworking the CI/CD page as well, in time for the upcoming alpha release?
* A TW review of this specific PR only (for the upcoming alpha release?), with a view to separately updating the CI/CD page in a different/new PR?

The second option seems the most logical, but I couldn't quite tell from the comments history 😊

**Comment by mesellings** [comment]
Thanks for the clarification @JacobiCamunda, I can take a look at this for you this week 👍

**Review by mesellings — APPROVED** [review]
@JacobiCamunda I've made changes directly to the branch, and added in a caution for that multiple process apps issue - happy to approve, but you might want to check the caution is correct 👍

**Comment by JacobiCamunda** [comment]
Thanks, @mesellings. Perfect!

**Pull request merged**